frontend/src/employee-frontend/components/unit/tab-groups/groups/Group.tsx

```diff
@@ -604,13 +607,22 @@ export default React.memo(function Group({
                           </FixedSpaceRow>
                         </Td>
                         <Td data-qa="placement-type">
-                          <CareTypeChip
-                            type={
-                              'type' in placement
-                                ? placement.type
-                                : 'backup-care'
-                            }
-                          />
+                          <FixedSpaceColumn
+                            spacing="xs"
+                            alignItems="flex-start"
+                          >
+                            <CareTypeChip
+                              type={
+                                'type' in placement
+                                  ? placement.type
+                                  : 'backup-care'
+                              }
+                            />
+                            {'fromUnits' in placement &&
+                              placement.fromUnits.map((unit) => (
+                                <Light key={unit.id}>{unit.name}</Light>
+                              ))}
```

Comment:
`'fromUnits' in placement` (and also above in `'type' in placement`) seems hacky. Could the types be enhanced?

Reply:
No idea